.gitignore

```diff
@@ -274,3 +274,4 @@ paket-files/
 # JetBrains Rider
 .idea/
 *.sln.iml
+*.targets
```

Comment:
These files are part of the project and should not be ignored. The local `DevVars.targets` file is already explicitly ignored.